Resources/views/CRUD/edit_orm_many_to_many.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% if sonata_admin.field_description.hasassociationadmin %}
-    <div id="field_container_{{ id }}" class="field-container">
+    <div id="field_container_{{ id }}" class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
```

Comment:
There are longer lines in the same file - is there a standard to what the length of a line should be in terms of characters?

Will this PR get blocked because the line looks long?